Firebase.cpp

```diff
@@ -97,18 +105,22 @@ bool Firebase::available() {
   return _http.getStreamPtr()->available();
 }
 
-Firebase::Event Firebase::read(String& event) {
+Firebase& Firebase::read() {
   auto client = _http.getStreamPtr();
-  Event type;;
-  String typeStr = client->readStringUntil('\n').substring(7);
-  if (typeStr == "put") {
-    type = Firebase::Event::PUT;
-  } else if (typeStr == "patch") {
-    type = Firebase::Event::PATCH;
-  } else {
-    type = Firebase::Event::UNKNOWN;
-  }
-  event = client->readStringUntil('\n').substring(6);
+  _event = client->readStringUntil('\n').substring(7);
+  _data = client->readStringUntil('\n').substring(6);
   client->readStringUntil('\n'); // consume separator
-  return type;
+  Serial.println(_event);
+  Serial.println(_data);
+  return *this;
+}
+
+JsonObject& Firebase::create() {
+  return _json.createObject();
+}
+
+JsonObject& Firebase::json() {
+  _json = StaticJsonBuffer<200>();
```

Comment:
Also `json()` is miss-leading because it's not clear that it will work if the user call it two times in a row (since `parseObject` does mutate the string buffer to add `\0`).